examples/1-with-urql-client/pages/index.tsx

```diff
@@ -2,25 +2,26 @@ import React from 'react';
 import Head from 'next/head';
 import { withUrqlClient } from 'next-urql';
 
-import PokemonList from '../components/pokemon_list';
+import { NextContext } from '../../../node_modules/@types/next/index';
```

Comment:
Definitely agree that we should upgrade our types to be compatible with the versions of Next we're using here. IIRC there wasn't a v9 available when we started, but that now that there is we should use it. Good shout!